services/figgy.md

```diff
@@ -100,4 +100,36 @@ Sometimes the `simple-tiles` library and the related `simpler-tiles` gem need to
   $ sudo make install
   $ cd .. && sudo rm -r simple-tiles-0.6.0/
   ```
-1. Redeploy Figgy to the server using Capistrano or Pulbot.
\ No newline at end of file
+1. Redeploy Figgy to the server using Capistrano or Pulbot.
+
+## Exporting Large Objects to Disk
+
+When we need to share a large number of full-resolution files with a user, exporting all the files
+attached to an object scales better than manually downloading each file. Use the
+`figgy:export:files` Rake task to export an item, which also includes files attached to child
+objects of MVWs. For example, for the Figgy object
+https://figgy.princeton.edu/catalog/5a6e59c2-8b8d-4a70-bc6c-cad38e781636 has the
+`source_metadata_identifier` C1384_c0289. To export this oobject, login to one of the lib-proc
+machines as the `deploy` user and run the Rake task:
```

Comment:
```suggestion
To export a large number of files related to a single object, use the `figgy:export:files` rake task 
instead of manually downloading each file. The rake task will export all files, including files attached
to the child objects of multi-volume works. For example, for the Figgy object
https://figgy.princeton.edu/catalog/5a6e59c2-8b8d-4a70-bc6c-cad38e781636 has the
`source_metadata_identifier` C1384_c0289. To export this object, login to one of the lib-proc
machines as the `deploy` user and run the Rake task:
```